upload/docker_images/validator/validator_harness.py

```diff
@@ -112,20 +112,19 @@ def _run_validator(self):
             'exception': None
         }
         try:
+            _subprocess_filename = "SUBPROCESS_LOG_FILE"
```

Comment:
The log_filename should probably include something unique about the request? Many validators will be running on a single machine and would stamp on each others log files